doc/api/child_process.markdown

```diff
@@ -157,6 +157,13 @@ the exit code of the child process while `error.signal` will be set to the
 signal that terminated the process. Any exit code other than `0` is considered
 to be an error.
 
+The `stdout` and `stderr` arguments passed to the callback will contain the
+stdout and stderr output of the child process. By default, Node.js will decode
+the output as UTF-8 and pass strings to the callback. The `encoding` option
+(see below) can be used to specify the character encoding used to decode the
```

Comment:
I would either link to `encoding` or just leave off the `(see below)`. These things tend to become wrong when things get moved around.